### ISSUE TITLE
Phone not Transferring audio to headphone (Probably because there is an option in the developer option under bluetooth audio that is greyed out)

### ISSUE THREAD

**Issue by ICED1089** [MisterZtr/PixelOS_gsi#102]
The Disable Bluetooth LE audio hardware offload and Disable Bluetooth A2DP hardware offload is enabled and is greyed out. I did a bit of research and I guess this option is for higher model headphones and not for lower end ones.
Because of this I can connect to my headphone **but as soon as i start playing media a white noise plays in my headphones and the phone disconnects it.** I tried to disable developer and enable. The grey option went and when I turned it off it told me to reboot and after reboot its greyed out again
Phone- Infinix Zero 8
Rom- Pixel OS GSI build latest July update

Almost everything is working fine and quite confident those two greyed option is the problem

**Comment by abrhst** [comment]
> The Disable Bluetooth LE audio hardware offload and Disable Bluetooth A2DP hardware offload is enabled and is greyed out. I did a bit of research and I guess this option is for higher model headphones and not for lower end ones. Because of this I can connect to my headphone **but as soon as i start playing media a white noise plays in my headphones and the phone disconnects it.** I tried to disable developer and enable. The grey option went and when I turned it off it told me to reboot and after reboot its greyed out again Phone- Infinix Zero 8 Rom- Pixel OS GSI build latest July update
> 
> Almost everything is working fine and quite confident those two greyed option is the problem

Check in phh treble settings and there should be an option for bluetooth fixes

**Comment by ICED1089** [comment]
> > The Disable Bluetooth LE audio hardware offload and Disable Bluetooth A2DP hardware offload is enabled and is greyed out. I did a bit of research and I guess this option is for higher model headphones and not for lower end ones. Because of this I can connect to my headphone **but as soon as i start playing media a white noise plays in my headphones and the phone disconnects it.** I tried to disable developer and enable. The grey option went and when I turned it off it told me to reboot and after reboot its greyed out again Phone- Infinix Zero 8 Rom- Pixel OS GSI build latest July update
> > Almost everything is working fine and quite confident those two greyed option is the problem
> 
> Check in phh treble settings and there should be an option for bluetooth fixes

Yes Tried the Bluetooth workarounds, still does not transfer audio to headphone. But it does transfer audio to my home speaker system but not the headphones :(

**Comment by abrhst** [comment]
> > > The Disable Bluetooth LE audio hardware offload and Disable Bluetooth A2DP hardware offload is enabled and is greyed out. I did a bit of research and I guess this option is for higher model headphones and not for lower end ones. Because of this I can connect to my headphone **but as soon as i start playing media a white noise plays in my headphones and the phone disconnects it.** I tried to disable developer and enable. The grey option went and when I turned it off it told me to reboot and after reboot its greyed out again Phone- Infinix Zero 8 Rom- Pixel OS GSI build latest July update
> > > Almost everything is working fine and quite confident those two greyed option is the problem
> > 
> > 
> > Check in phh treble settings and there should be an option for bluetooth fixes
> 
> Yes Tried the Bluetooth workarounds, still does not transfer audio to headphone. But it does transfer audio to my home speaker system but not the headphones :(

Hmm, maybe the headphones are defective?

**Comment by ICED1089** [comment]
> > > > The Disable Bluetooth LE audio hardware offload and Disable Bluetooth A2DP hardware offload is enabled and is greyed out. I did a bit of research and I guess this option is for higher model headphones and not for lower end ones. Because of this I can connect to my headphone **but as soon as i start playing media a white noise plays in my headphones and the phone disconnects it.** I tried to disable developer and enable. The grey option went and when I turned it off it told me to reboot and after reboot its greyed out again Phone- Infinix Zero 8 Rom- Pixel OS GSI build latest July update
> > > > Almost everything is working fine and quite confident those two greyed option is the problem
> > > 
> > > 
> > > Check in phh treble settings and there should be an option for bluetooth fixes
> > 
> > 
> > Yes Tried the Bluetooth workarounds, still does not transfer audio to headphone. But it does transfer audio to my home speaker system but not the headphones :(
> 
> Hmm, maybe the headphones are defective?

They worked on the stock rom and are still working with my laptop and iPhone. The problem is the "greyed out" feature under developer option "disable Bluetooth a2dp hardware offload" i cant turn it of because it is greyed out and I watch a video saying to turn this off if there is connectivity issues
